examples/vision/mixUp.py

```diff
@@ -0,0 +1,226 @@
+"""
+Title: mixup augmentation for image classification
+Author: [Sayak Paul](https://twitter.com/RisingSayak)
+Date created: 2021/03/06
+Last modified: 2021/03/06
+Description: Data augmentation using the mixup technique for image classification.
+"""
+"""
+## Introduction
+"""
+
+"""
+mixup is a *domain-agnostic* data augmentation technique proposed in [mixup: Beyond Empirical Risk Minimization](https://arxiv.org/abs/1710.09412) 
```

Comment:
If you aren't going to capitalize the name of the technique, at least use italics like in the paper.